### ISSUE TITLE
Starting a poll crashes HTML5 client

### ISSUE THREAD

**Issue by florianjanus** [bigbluebutton/bigbluebutton#12149]
**Describe the bug**
When starting a new poll, **sometimes** the GUI of the html5 client disappears (only the blue background remains visible). The only way to get everything back seems to be a page reload.

**To Reproduce**
1. As a moderator prepare a new poll (type does not seem to matter) in a running conference with at least one other participant.
2. Click on the "Start poll" button.
3. If the poll starts as expected, close it and repeat steps 1 & 2.

**Expected behavior**
The polls all reliably start as expected.

**Actual behavior**
In general, the first few polls in a conference seem to always work without issues. But after a few polls, the html5 client crashes when clicking the "Start poll" button. Sometimes it's the 5th poll that crashes, sometimes it's the 25th poll that crashes... At the moment of the crash, error messages appear in the browser console (cf. screenshot below).

**Screenshots**
![bbb-poll-crash](https://user-images.githubusercontent.com/16020573/116007965-a010a680-a612-11eb-8578-019ab3062f2c.png)

**BBB version (optional):**
We have observed the bug with various 2.3 alpha/beta versions and also in 2.3-rc-1 (HTML5 client build: 1643).

**Desktop:**
 - OS: Bug has been observed by multiple users running Windows 10, various versions of MacOS and Linux.
 - Browser: We have been able to reproduce the bug using Firefox 88 and Chromium 90.

**Additional context**
The server logs don't seem to contain any related error messages. Running bbb-conf --check doesn't list any "potential problems".

Based on the bug description, #11167 might be related. However the bug described above also occurs with just two people in the conference (in contrast to the 15+ people mentioned in #11167).

**Comment by ichdasich** [comment]
Hrm, is this on RC1 or beta-5? 

I have user reports of the same form for beta-5, but was hoping that it was the issue fixed in #12081; This seems to be something else?

**Comment by florianjanus** [comment]
I admit this seems to be somewhat of a heisenbug.
It's been hard to investigate the exact circumstances that cause this kind of crash as the error may occur in one session, but then may not be reproducible even after 100+ polls in another session ...and then reoccur in some other session (all of this with identical server and client configs).

Based on our internal user feedback it seems that indeed most poll crashes in the past have since been fixed by #12081. However we still encountered the error a couple of times after updating to RC1 and in scenarios that differ from #12075:

- with just two moderators in the conference, which did not leave during the poll
- with polls that crashed instantly before anybody could respond to them (all previous polls in that session worked as expected).

**Comment by basisbit** [comment]
@florianjanus please provide the browser developer console log as well as the bbb-html5/NodeJS log. Thanks.

**Comment by antobinary** [comment]
Thanks for reporting! I have not spotted this crash before but @ramonlsouza will try to narrow down the case when it's happening based on the screenshot you sent.
Please provide any further info you may have about this to help accelerate the fix. In particular some more ideas on when it could be happening, client side logs as already mentioned...

**Comment by ramonlsouza** [comment]
I was able to reproduce it by following these steps:

1. enable popup alerts
2. start new poll
3. publish results (without leaving poll panel)

repeat 2 and 3 and the crash should happen in a few tries

**Comment by ichdasich** [comment]
Can reproduce it with that. Without enabling pop-up alerts, however, (default, as it seems) there was no crash.

**Comment by florianjanus** [comment]
I agree that the error seems to be connected to the pop-up alerts. In all cases where we encountered a crash, all pop-up alerts were enabled. With the default pop-up alert settings I haven't been able to reproduce the crash.

I've attached the full browser console log as requested. The NodeJS log I'll have to track down later today.

[bbb_poll_crash_brower_console.log](https://github.com/bigbluebutton/bigbluebutton/files/6378239/bbb_poll_crash_brower_console.log)
